main/src/ca/uwaterloo/flix/language/errors/ResolutionError.scala

```diff
@@ -162,7 +162,29 @@ object ResolutionError {
       vt << NewLine
       vt << Code(loc, "illegal type application.")
       vt << NewLine
```

Comment:
Could we say that:

The type constructor has kind '$kind' and the argument has kind '$kind'?